fe/src/main/java/org/apache/doris/system/SystemInfoService.java

```diff
@@ -443,7 +390,7 @@ public void releaseBackends(String clusterName, boolean isReplay) {
         }
 
         List<List<Backend>> hostList = Lists.newArrayList(hostBackendsMapInCluster.values());
-        Collections.sort(hostList, hostBackendsListComparator);
+        hostList.sort(hostBackendsListComparator);
```

Comment:
Actually, the method `calculateDecommissionBackends()` is deprecated.
So could you please add `@Derepcated` to this method?

Reply:
done